cmd/create/cluster/cmd.go

```diff
@@ -508,6 +511,15 @@ func init() {
 			"roles in STS clusters.",
 	)
 
+	// Options releated to HyperShift:
+	flags.BoolVar(
```

Comment:
Do we need interactive mode for this

Reply:
we should have interactive support

Reply:
Good call. Will add the support for the interactive mode